nav2_system_tests/src/updown/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+add_executable(test_updown
```

Comment:
Do you plan to make the test executable by Ctest in the future, or is this only intended to be run manually?

Reply:
I've been launching it manually to let it run overnight or over the weekend. I don't think it needs to be part of our normal CI process, but in the future we could have some automated stress tests. Not sure if we want or need those to be gtest-enabled.